tests/ui/consts/const_in_pattern/issue-34784-match-on-non-int-raw-ptr.rs

```diff
@@ -23,10 +23,18 @@ fn foo2(x: *const u8) {
 
 const D: *const [u8; 4] = b"abcd";
 
+const STR: *const str = "abcd";
+
 fn main() {
     match D {
         D => {} //~ERROR: behave unpredictably
         //~| previously accepted
         _ => {}
     }
+
+    match C {
+        C => {} //~ERROR: behave unpredictably
+        //~| previously accepted
+        _ => {}
+    }
 }
```

Comment:
is it intended that the var is called `STR` but inside the match you use `C`? (should that have been `STR`as well?)

Reply:
*oops*